src/main/java/uk/gov/hmcts/opal/dto/AccountSearchDto.java

```diff
@@ -29,4 +30,11 @@ public class AccountSearchDto implements ToJsonString {
     private String majorCreditor;
     /** Unsure. */
     private String tillNumber;
+
+    public Long getNumericCourt() {
```

Comment:
How about  if we return `Optional<Long>` instead returning `null` t line 38 ?

Reply:
I did think about doing that, but in the places where this is being utilised, I actually want a null.